manager/logbroker/broker.go

```diff
@@ -41,11 +41,32 @@ func New() *LogBroker {
 	}
 }
 
+// Run the log broker
+func (lb *LogBroker) Run(ctx context.Context) error {
+	lb.mu.Lock()
+	lb.stopped = make(chan struct{})
+	lb.logQueue = watch.NewQueue()
+	lb.subscriptionQueue = watch.NewQueue()
```

Comment:
If you create the queues here, you shouldn't create them in `New`. Otherwise there's a leak.